city_scrapers/spiders/losca_Health_Commission.py

```diff
@@ -0,0 +1,104 @@
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class LoscaHealthCommissionSpider(CityScrapersSpider):
+    name = "losca_Health_Commission"
+    agency = "Los Angeles Health Commission"
+    timezone = "America/Los_Angeles"
+    committee_id: int = 6
+    start_year: int = 2024
+    default_location = {
+        "address": "200 N Spring St, Room 340 (CITY HALL) Los Angeles, CA 90012",
+        "name": "Los Angeles City Health Commission",
+    }
```

Comment:
Can you update "address" of the `default_location` so that it would be parsed properly when being imported into our Documenters.org system (move the text "Room 340 (CITY HALL)" to the beginning of "address"
```
default_location = {
    "address": "Room 340 (CITY HALL), 200 N Spring St, Los Angeles, CA 90012"",
    "name": "Los Angeles City Health Commission",
}
```